packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -374,7 +374,7 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.widgets.edtiableText.scrollPhysics}
   final ScrollPhysics scrollPhysics;
 
-  /// {@macro flutter.dart:ui.Text.TextWidthBasis}
+  /// Defines how to consider the width of the rendered text.
```

Comment:
consider => measure

I think the TextWidthBasis docs also use "consider"; doesn't seem like the right word to me.